scss/_card.scss

```diff
@@ -7,7 +7,7 @@
   display: flex;
   flex-direction: column;
   background-color: $card-bg;
-  border: $card-border-width solid $card-border-color;
+  box-shadow: $card-shadow;
```

Comment:
Agree on the other ones, we can do that. This here is a special one. This would mean we need to create two new variables (for borders of tables and list groups) and null the $card-border-width and $card-border-color. If you'd rather do that, we can do that.